test/pbcross.go

```diff
@@ -0,0 +1,46 @@
+// `go get github.com/ipfs/go-merkledag`
```

Comment:
I'm not sure if this should be in the `test` directory. Perhaps we create a new directory called `tools` ([js-multicodec also has a tools directory](https://github.com/multiformats/js-multicodec/tree/86a6b84ce176fed29b2e1d3f7698e40f6fb8dc79/tools) (or any better name you come up with) to store this file, so people don't wonder why there's a Go source file in their test directory.

Or if those fixtures are already somewhere else (e.g. in go-car), we could just put a link to it in the js test and interested people can go from there.

Reply:
moved to tools. go-car doesn't separate them out like this, I figure it's nice to have a proper reference so if we come across something new that's broken between the two implementations we can add it to the .go implementation